doc/rtd/topics/tutorials/qemu.rst

```diff
@@ -0,0 +1,140 @@
+.. _tutorial_qemu:
+
+Qemu
+****
+
+In this tutorial, we will create our first cloud-init user data config and
```

Comment:
I'd rework this paragraph a bit to first briefly explain what Qemu is and why someone might want to use it for deploying cloud-init, and then telling the user what end goal the tutorial will walk them towards.

Reply:
+1